### ISSUE TITLE
Breaks when used with dragonfly

### ISSUE THREAD

**Issue by christianlerke** [wvanbergen/request-log-analyzer#179]

**Comment by christianlerke** [comment]
Dragonfly generates @tempfile in the log

which throws the following error

```
/shared/bundle/ruby/2.1.0/gems/request-log-analyzer-1.13.1/lib/request_log_analyzer/request.rb:40:in `eval': (eval):1: syntax error, unexpected tCONSTANT, expecting '}' (SyntaxError)
...0000542aa90 @tempfile="Tempfile:/tmp/RackMultipart20141031-1...
...                ^
```

**Comment by wvanbergen** [comment]
This should be fixed in the latest version, 1.13.4
